config/runtime.exs

```diff
@@ -52,7 +63,7 @@ if config_env() == :prod and splunk_token != "" do
     connector: Logger.Backend.Splunk.Output.Http,
     host: 'https://http-inputs-mbta.splunkcloud.com/services/collector/event',
     token: splunk_token,
-    format: "$dateT$time [$level]$levelpad node=$node $metadata$message\n",
+    format: "$dateT$time [$level] node=$node $metadata$message\n",
```

Comment:
`$levelpad` was deprecated with this PR https://github.com/elixir-lang/elixir/pull/11633